releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,580 @@
+    - Support a new built-in function `CHARSET()` [#3931](https://github.com/pingcap/tidb/issues/3931)
+    - Support filtering a baseline capturing blocklist by usernames [#32558](https://github.com/pingcap/tidb/issues/32558)
+    - Support using wildcards in a baseline capturing blocklist [#32714](https://github.com/pingcap/tidb/issues/32714)
+    - Optimize the results of the `ADMIN SHOW DDL JOBS` and `SHOW TABLE STATUS` statements by displaying the time according to the current `time_zone` [#26642](https://github.com/pingcap/tidb/issues/26642)
+    - Support pushing down the `DAYNAME()` and `MONTHNAME()` functions to TiFlash [#32594](https://github.com/pingcap/tidb/issues/32594)
+    - Support pushing down the `REGEXP` function to TiFlash [#32637](https://github.com/pingcap/tidb/issues/32637)
+    - Support pushing down the `DAYOFMONTH()` and `LAST_DAY()` functions to TiFlash [#33012](https://github.com/pingcap/tidb/issues/33012)
+    - Support pushing down the `DAYOFWEEK()` and `DAYOFYEAR()` functions to TiFlash [#33130](https://github.com/pingcap/tidb/issues/33130)
+    - Support pushing down the `IS_TRUE`, `IS_FALSE`, and `IS_TRUE_WITH_NULL` functions to TiFlash [#33047](https://github.com/pingcap/tidb/issues/33047)
+    - Support pushing down the `GREATEST` and `LEAST` functions to TiFlash [#32787](https://github.com/pingcap/tidb/issues/32787)
+    - Support tracking the execution of the `UnionScan` operator [#32631](https://github.com/pingcap/tidb/issues/32631)
+    - Support using the PointGet plan for queries that read the `_tidb_rowid` column [#31543](https://github.com/pingcap/tidb/issues/31543)
+    - Support showing the original partition name in the output of the `EXPLAIN` statement without converting the name to lowercase [#32719](https://github.com/pingcap/tidb/issues/32719)
+    - Enable partition pruning for RANGE COLUMNS partitionings on IN conditions and string type columns [#32626](https://github.com/pingcap/tidb/issues/32626)
+    - Return an error message when a system variable is set to NULL [#32850](https://github.com/pingcap/tidb/issues/32850)
+    - Remove Broadcast Join from the non-MPP mode [#31465](https://github.com/pingcap/tidb/issues/31465)
+    - Support executing MPP plans on partitioned tables in dynamic pruning mode [#32347](https://github.com/pingcap/tidb/issues/32347)
+    - Support pushing down predicates for common table expressions (CTEs) [#28163](https://github.com/pingcap/tidb/issues/28163)
+    - Simplify the configurations of `Statement Summary` and `Capture Plan Baselines` to be available on a global basis only [#30557](https://github.com/pingcap/tidb/issues/30557)
+    - Update gopsutil to v3.21.12 to address alarms reported when building binary on macOS 12 [#31607](https://github.com/pingcap/tidb/issues/31607)
+
++ TiKV
+
+    - Improve the Raftstore sampling accuracy for large key range batches [#12327](https://github.com/tikv/tikv/issues/12327)
```

Comment:
Improve the Raftstore sampling accuracy for batches with many key ranges [#12327](https://github.com/tikv/tikv/issues/12327)